app/views/users/edit.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= form_with model: @user, local: true do |form| %>
+  <div class="avatar-img">
+    <img src="<%= @user.avatar.profile.url if @user.avatar? %>", id='avatar'>
+  </div>
+  <div class="columns" style="margin-top: 5px;">
+    <div class="column is-1">
+      <div class="avatar-upload">
+        <label for="file-input">
+          <span class="button">Upload avatar</span>
+        </label>
+        <%= form.file_field :avatar, class: 'button', id: 'file-input', onchange: "onFileSelected(event)" %>
+      </div>
+    </div>
+    <div class="column is-1" style="margin-left: 40px;">
+      <div class="avatar-delete">
+        <label for="avatar-delete-button">
+          <span class="button">Delete avatar</span>
+        </label>
+      </div>
+    </div>
+  </div>
+  <div class="columns">
+    <div class="column is-4">
+      <%= form.label :nickname, class: 'label' %>
+      <%= form.text_field :nickname, class: 'input' %>
+
```

Comment:
How about first name, last name, email, since we have them and can actually be able to edit